### PR TITLE
Update sysact

### PR DESCRIPTION
**Pull request by wickedjargon**
this change removes the incorrect white space char next to `shutdown` and `renew` and replaces it with a regular white space char.

The change is not fully reflected in the diff that github shows. I think github identifies both white space chars as the same char.

You're probably thinking "WTF is this guy talking about". If so, please replicate the issue by changing the default font / dmenu's font to Liberation Mono and hit Mod+BackSpace. You should see box chars next to  `shutdown` and `renew`.

### CODE REVIEW

**Comment by kronikpillow** [comment]
this is more of a issue to fontconfig and font settings ... I'v had that white space bar next to the sysact shutdown, also on many emoji ... but the issue is gone since i installed other fonts like 

ttf-anonymous-pro
ttf-bitstream-vera
ttf-croscore
ttf-dejavu
ttf-droid
ttf-fira-code
ttf-fira-mono
ttf-fira-sans
ttf-font-awesome
ttf-hack
ttf-inconsolata
ttf-iosevka-nerd
ttf-jetbrains-mono
ttf-joypixels
ttf-liberation
ttf-linux-libertine
ttf-mononoki
ttf-nerd-fonts-symbols
ttf-roboto
ttf-roboto-mono
ttf-ubuntu-font-family

...

**Comment by wickedjargon** [comment]
I believe my commit would fix the issue for all fonts.

**Comment by narukeh** [comment]
TLDR: My opinion is that this PR should be rejected.
@wickedjargon  have a read:
https://github.com/LukeSmithxyz/voidrice/pull/753
and my comments in https://github.com/LukeSmithxyz/voidrice/pull/972
https://github.com/LukeSmithxyz/voidrice/issues/844#issuecomment-735734194
https://codepoints.net/U+FE0F

Some emojis have an additional "glyph" (which do not render), so they cam modify/manipulate whatever...
e.g.:the [`<200d>`](https://emojipedia.org/zero-width-joiner/) combines two emojis. Heart on fire = heart + fire.

**long read ahead!**
replication: (this is a recurring theme) also happening e.g. with the single eye emoji
using nvim, write two lines, so you can see the difference, on the first line write "eyeEYEEMOJIeye" with no spaces: `eye👁️eye`
and on the second line, write "eye EYEEMOJI eye" with two spaces `eye 👁️ eye`
then save and `cat` the file to the terminal. You will see that the second line only has one space before the emoji, and that in the first line the second word eye gets written above the emoji.
innvim
inst
So, yeah i have put a space between the PC-emji and _shutddown_, but because how the emoji behaves, it "eats" the next character, so if you reeeally care how it looks, then yeah you should two spaces. (PC-emojiSPACESPACEshutdown)

**Comment by wickedjargon** [comment]
With your explanation, it is clear you understand the issue better. I'm ok if this PR is rejected.

That being said, I just ran default LARBS (default dotfiles and progs file) and I still get the - boxes myself *with the default font and other settings*. Perhaps the issue is specific to my computer only.

**Comment by narukeh** [comment]
@wickedjargon No i get the same result, not just you (with defaults and heavily modified system). What i meant is that, its just **looks**, but just make sure you actually did it right, since it can not be seen here in github 😄. Just wanted to explain what is happening.

When i wrote the emoji file there was no way of removing the `<fe0f>` tag. (but i did manage to remove the other ones)

Using just echo in the terminal i think explains it better, be sure to remove that `<fe0f>` tag
info for screenshot: first two lines dont have any spaces, last two lines have two spaces but only one gets shown. Lines 3 and 4 dont seem to have any difference hmmmmm
pic-selected-211219-2123-20

🤷🤷🤷 whatever 😄😄

**Comment by kronikpillow** [comment]
like i said ... install all the above fonts i posted ... issue gone, emojis render regularly

**Comment by narukeh** [comment]
@kronikpillow well the point was to know what was going on. But i did install `fft-hack`, though nothing happened, same result. And installing **every** font you listed is not a solution. The solution (i think) is to replace `<fe0f>` with two spaces, so it looks "nice", like i have explained.

**Comment by wickedjargon** [comment]
Would you like to submit a commit for it?

**Comment by kronikpillow** [comment]
saying this for the 3rd tiem :) i installed all the above fonts 1 by 1, i'm not entirely sure which one solved it, maybe you can try going 1 by 1 with the listed fonts, update font-cache with fc-cache -fv after every installation, refresh dwm, and see what happens till it magically fixes, then we can find out what actually caused the fix ... I am to lazy to uninstall every single font i ever installed simply to debug something that you can do yourselves by installing fonts 1 by 1 ... 

this worked for me, with every single reinstall of LARBS 

or don't ...

**Comment by wickedjargon** [comment]
And what if the user decides to delete that font? I think we should choose emojis that render properly regardless of which fonts are installed. 

I would back @narukeh solution, but please submit the commit yourself.

**Comment by kronikpillow** [comment]
submit a commit to install fonts? 🤣🤣🤣 jeez, forget i said anything, I was just offering you a guaranteed working solution, your thing if you use it or not 👍🏻

**Comment by wickedjargon** [comment]
Yes, the prog.csv can be updated. That would be a commit

**Comment by kronikpillow** [comment]
dude ... I know what a commit is ... i contributed to voidrice multiple times, why does anyone have to create a commit, for you to install fonts?

**Comment by wickedjargon** [comment]
I am not using voidrice. My purpose was to improve LARBS, not seek help for my personal config.

**Comment by kronikpillow** [comment]
Ok ... so I have uninstalled all my  fonts ... then I'v went 1 by 1 reinstalling them, and figured out that it is ttf-dejavu that fixes the issue ... once a user installs ttf-dejavu, the white space is  gone both from emoji script and sysact script ... so ... it's something related to ttf-dejavu in /etc/fonts/conf.d but I am unable to figure out what ... if your able to investigate this, plz report back so that we can figure it out why dejavu fixes this problem ... if your interested to find out that is ...

**Comment by narukeh** [comment]
@kronikpillow hmmm that's interesting. I have always used dejavu as my fallback font, just recently i switched to noto, (__but__ i always have dejavu installed). I always had the "bug" white space, when i was using dejavu, and now that im using noto.
This is how my fontconfig loos like:
`cat .config/fontconfig/fonts.conf`
```xml
<?xml version='1.0'?>
<!DOCTYPE fontconfig SYSTEM 'fonts.dtd'>
<fontconfig>
        <alias>
                <family>serif</family>
                <prefer>
                        <family>Noto Serif</family>
                        <family>Noto Color Emoji</family>
                </prefer>
        </alias>
        <alias>
                <family>sans-serif</family>
                <prefer>
                        <family>Lexend Deca</family>
                        <family>Noto Color Emoji</family>
                </prefer>
        </alias>
        <alias>
                <family>sans</family>
                <prefer>
                        <family>Lexend Deca</family>
                        <family>Noto Color Emoji</family>
                </prefer>
        </alias>
        <alias>
                <family>monospace</family>
                <prefer>
                        <family>Terminus</family>
                        <family>Noto Sans Mono</family>
                </prefer>
        </alias>
</fontconfig>
```
At least now it got interesting, why its working for you but not for me, i would like to solve that mystery 😄😄
I'll tinker a bit, with different variations and let u know if anything happens.

**Comment by narukeh** [comment]
Well as for my experience it is not ttf-dejavu. Like i said i always have that one installed. I even tried this: (also log out&in again)
`cat ~cf/fontconfig/fonts.conf`
```xml
<?xml version='1.0'?>
<!DOCTYPE fontconfig SYSTEM 'fonts.dtd'>
<fontconfig>
        <alias>
                <family>serif</family>
                <prefer>
                        <family>DejaVu Serif</family>
                        <family>JoyPixels</family>
                </prefer>
        </alias>
        <alias>
                <family>sans-serif</family>
                <prefer>
                        <family>DejaVu Sans</family>
                        <family>JoyPixels</family>
                </prefer>
        </alias>
        <alias>
                <family>sans</family>
                <prefer>
                        <family>DejaVu Sans</family>
                        <family>JoyPixels</family>
                </prefer>
        </alias>
        <alias>
                <family>monospace</family>
                <prefer>
                        <family>DejaVu Sans Mono</family>
                        <family>JoyPixels</family>
                </prefer>
        </alias>
</fontconfig>
```
still the same result as before.
